src/libraries/System.IO/tests/BinaryWriter/BinaryWriter.EncodingTests.cs

```diff
@@ -191,36 +195,35 @@ public void WriteString_NotUtf8(int stringLengthInChars)
         [SkipOnPlatform(TestPlatforms.Android, "OOM on Android could be uncatchable & kill the test runner")]
         public unsafe void WriteChars_VeryLargeArray_DoesNotOverflow()
```

Comment:
Should it be outer loop as well?  I believe many of our "allocate huge amounts of memory" tests are.

Reply:
Good idea, I am going to move it to outerloop